visidata/canvas.py

```diff
@@ -552,11 +554,15 @@ def zoomTo(self, bbox):
         self.xzoomlevel=bbox.w/self.canvasBox.w
         self.yzoomlevel=bbox.h/self.canvasBox.h
 
+        self.resetBounds()
```

Comment:
They can't be combined, as each is currently called without the other in at least one place.

`resetBounds()` and some of the nearby code would benefit from a refactoring and reordering. That would allow the two lines to be combined. But it'd be a bigger change, and it'd belong in a separate PR.

Reply:
I take it back: on thinking about it some more, I think `resetBounds()` and `refresh()` can be combined. I'll update this PR to do that.